packages/liveblocks-react-ui/src/slate/utils/is-selection-collapsed.ts

```diff
@@ -1,8 +0,0 @@
-import type { Selection as SlateSelection } from "slate";
-import { Range as SlateRange } from "slate";
-
-export function isSelectionCollapsed(
```

Comment:
This type guard was causing more harm than good by combining the null check and the collapsed one.